code/json.sls

```diff
@@ -30,172 +30,218 @@
 
 (library (json)
   (export json-write
-	  json-read)
-  
+          json-read)
+
   (import
-    (except (rnrs) define-record-type error)
-    (rnrs r5rs)
-    (rnrs mutable-pairs)
-    (only (srfi :1 lists) lset-union append-map fold)
-    (srfi :6 basic-string-ports)
-    (srfi :9 records)
-    (srfi :23 error)
-    (only (chezscheme) include)
-    (packrat))
+   (except (rnrs) define-record-type error)
+   (rnrs r5rs)
+   (rnrs mutable-pairs)
+   (only (srfi :1 lists) lset-union append-map fold)
+   (srfi :6 basic-string-ports)
+   (srfi :9 records)
+   (srfi :23 error)
+   (only (chezscheme) include)
+   (packrat))
 
+  ;; Helper to convert from kebab-case to snake_case
```

Comment:
I wasn't sure where this helper should go.  It is useful, so I don't imagine we want it buried inside `json-write`.